e2e/whisper/whisper_mailbox_test.go

```diff
@@ -142,99 +105,222 @@ func (s *WhisperMailboxSuite) TestRequestMessageFromMailboxAsync() {
 					"to":` + strconv.FormatInt(time.Now().UnixNano()/int64(time.Second), 10) + `
 		}]
 	}`
-	resp = rpcClient.CallRaw(reqMessagesBody)
+	resp := rpcClient.CallRaw(reqMessagesBody)
 	reqMessagesResp := baseRPCResponse{}
 	err = json.Unmarshal([]byte(resp), &reqMessagesResp)
 	s.Require().NoError(err)
-	s.Require().Nil(postResp.Err)
+	s.Require().Nil(reqMessagesResp.Err)
 
 	//wait to receive message
 	time.Sleep(time.Second)
 	//And we receive message
-	resp = rpcClient.CallRaw(`{
-		"jsonrpc": "2.0",
-		"method": "shh_getFilterMessages",
-		"params": ["` + messageFilterID + `"],
-		"id": 1}`)
-
-	err = json.Unmarshal([]byte(resp), &messages)
-	//assert
-	s.Require().NoError(err)
-	s.Require().Equal(1, len(messages.Result))
+	messages = s.getMessagesByMessageFilterId(rpcClient, messageFilterID)
+	s.Require().Equal(1, len(messages))
 
 	//check that there are no messages
-	resp = rpcClient.CallRaw(`{
-		"jsonrpc": "2.0",
-		"method": "shh_getFilterMessages",
-		"params": ["` + messageFilterID + `"],
-		"id": 1}`)
-
-	err = json.Unmarshal([]byte(resp), &messages)
-	//assert
-	s.Require().NoError(err)
-	s.Require().Equal(0, len(messages.Result))
+	messages = s.getMessagesByMessageFilterId(rpcClient, messageFilterID)
+	s.Require().Equal(0, len(messages))
 
 	//Request each one messages from mailbox, using same params
 	resp = rpcClient.CallRaw(reqMessagesBody)
 	reqMessagesResp = baseRPCResponse{}
 	err = json.Unmarshal([]byte(resp), &reqMessagesResp)
 	s.Require().NoError(err)
-	s.Require().Nil(postResp.Err)
+	s.Require().Nil(reqMessagesResp.Err)
 
 	//wait to receive message
 	time.Sleep(time.Second)
 	//And we receive message
-	resp = rpcClient.CallRaw(`{
-		"jsonrpc": "2.0",
-		"method": "shh_getFilterMessages",
-		"params": ["` + messageFilterID + `"],
-		"id": 1}`)
+	messages = s.getMessagesByMessageFilterId(rpcClient, messageFilterID)
+	s.Require().Equal(1, len(messages))
+}
+
+func (s *WhisperMailboxSuite) TestRequestMessagesFromMailboxFromGroupChat() {
```

Comment:
Maybe `TestRequestMessagesInGroupChat` would be more concise?

Reply:
Yes, it's simpler. Thanks!